.buildkite_win/appxmanifest.xml

```diff
@@ -6,7 +6,7 @@
   xmlns:desktop="http://schemas.microsoft.com/appx/manifest/desktop/windows10">
   <Identity
     Name="EndlessOSFoundation.EndlessKey.app"
-    Version="0.1.10.1"
+    Version="0.1.11.0"
```

Comment:
The commit message says:

> the version format is "Major.Minor.Build.Revision",
> where "Major" can never be 0.

but Major here is 0. Don't you need to bump this to e.g. 1.0.0.0 ?

Reply:
Good catch, that completely slip through. I'm surprised the upload validation logic did not catch it. I'll bump it to 1.0.0.0 and update the commit message accordingly.

@danigm FYI, looks like this will app will need to be released as 1.0.0.0 (or greater) once we are ready to publish to the Microsoft Store, to satisfy the store requirements.